multispaeti/_multispati_pca.py

```diff
@@ -86,107 +89,123 @@ class MultispatiPCA:
     <https://onlinelibrary.wiley.com/doi/abs/10.3170/2007-8-18312>`_
     """
 
-    # TODO, should scaling be part of multispati
-
     def __init__(
         self,
         n_components: int | tuple[int, int] | None = None,
         *,
-        connectivity: sparray | spmatrix,
+        connectivity: _Connectivity | None = None,
     ):
-        self._fitted = False
-        W = csr_array(connectivity)
+        self.n_components = n_components
+        self.connectivity = connectivity
```

Comment:
maybe validate the connectivity and n_components at the time of initialisation

Reply:
I did that before, but it should not be done according to sklearn because in some pipelines these values might be updated after object creation

Reply:
strange but okay then